### ISSUE TITLE
TYPO3 10 compatibility

### ISSUE THREAD

**Issue by Chrisswald** [saschaende/t3helpers#5]
Hi Sascha,

do you plan to migrate your extension?

**Comment by saschaende** [comment]
Yes, this is definitely planned as soon as I migrate my website to TYPO3 10 (probably October/November). But I'm still grateful for hints about what needs to be changed.
